microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/security/publickeylocation/PublicKeyFileLocationTestCase.java

```diff
@@ -0,0 +1,35 @@
+package org.jboss.eap.qe.microprofile.jwt.security.publickeylocation;
+
+import org.jboss.arquillian.container.test.api.OperateOnDeployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunAsClient
+@RunWith(Arquillian.class)
+public class PublicKeyFileLocationTestCase {
```

Comment:
Why is this class here? Maybe future TD? Or is the PR still WIP?

Reply:
Will remove. Nice catch!